### ISSUE TITLE
Transfer Test-Infra Repository

### ISSUE THREAD

**Issue by BRMcLaren** [openenclave/openenclave#3618]
I need to transfer this repository https://github.com/openenclave-ci/test-infra/ over to the openenclave organization. It entails the decoupling of the open enclave test infrastructure, docker images and bot automation for PR's/labelling.

**Comment by dthaler** [comment]
As a test case for PR #3534... :) 
What are the proposed answers to the following questions (copied from that PR):

1. How should issues with the new repo be tracked and triaged? Should github issues be filed in the new repo, or in the main openenclave repo?  Should they be discussed in the master openenclave triage meeting, or in some separate process?
2. Should one or more of the existing SIGs (e.g., [SIG-Architecture](sig-architecture/README.md), [SIG-Testing](sig-testing/README.md)) oversee relevant aspects of the new repo, or not?
3. Who is reponsible for maintaining the new repo?  How is their contact info published?

**Comment by BRMcLaren** [comment]
1. In the short term, issues should be filed under the test-infra project individually until it sees adoption as the backend. Then it should be revisited by SIG-CI to what makes the most sense.
2. SIG-CI should oversee the relevant aspects of this repo.
3. The individuals responsible for maintaining this repo will be depicted by it's [owners]( https://github.com/openenclave-ci/test-infra/blob/master/OWNERS) list. This intends to be revisited every 90 days for activity from maintainers. I will make maintainers.md on the repo with owners' contact info (e-mail).

Clarification, for 3 will a DL be acceptable?

**Comment by radhikaj** [comment]
Radhika will send an email to CGC to see if anyone objects to transferring this repo.

**Comment by BRMcLaren** [comment]
@AevaOnline is helping me sort out #3 while we wait on e-mail response.

**Comment by AevaOnline** [comment]
For point 3
> Who is reponsible for maintaining the new repo?

For this specific case, I would suggest the use individual emails, because:

- the transparency is beneficial to the community, in that they may know who is on the team, and who to reach out to when there's a problem with the CI system
- also, for the individuals who are in that role, the reputation and connections gained are often part of the "reward" for accepting such a role in an open source project
- I would expect that integration with bot frameworks is much easier when the "source of truth" is a README or OWNERS file, rather than a membership list in some mail management system
- and lastly, it is one less thing to have to change when someone joins (or leaves) the team (since you have to have the OWNERS file anyway)

I think the first point (transparency) is a hard requirement (no team's member list should be opaque) though this could be handled in a number of ways, and a DL could still be used if that makes other things easier. I think the other points are just conveniences and nice-to-haves.

As a side point, Brett and I talked about using "oe-test-infra @ confidentialcomputing.io" as a distribution list for this, which I think could be viable, though I don't know if groups.io supports distribution lists or aliases (I'm only familiar with its support for discussion lists and managed groups). I'll ask around.

**Comment by radhikaj** [comment]
> For point 3
> 
> > Who is reponsible for maintaining the new repo?
> 
> For this specific case, I would suggest the use individual emails, because:
> 
> * the transparency is beneficial to the community, in that they may know who is on the team, and who to reach out to when there's a problem with the CI system
> * also, for the individuals who are in that role, the reputation and connections gained are often part of the "reward" for accepting such a role in an open source project
> * I would expect that integration with bot frameworks is much easier when the "source of truth" is a README or OWNERS file, rather than a membership list in some mail management system
> * and lastly, it is one less thing to have to change when someone joins (or leaves) the team (since you have to have the OWNERS file anyway)
> 
> I think the first point (transparency) is a hard requirement (no team's member list should be opaque) though this could be handled in a number of ways, and a DL could still be used if that makes other things easier. I think the other points are just conveniences and nice-to-haves.
> 
> As a side point, Brett and I talked about using "oe-test-infra @ confidentialcomputing.io" as a distribution list for this, which I think could be viable, though I don't know if groups.io supports distribution lists or aliases (I'm only familiar with its support for discussion lists and managed groups). I'll ask around.

Having individual owners does seem to make sense based on these arguments.

**Comment by BRMcLaren** [comment]
@dthaler @AevaOnline is a github alias an agreed-upon and accepted contact method?

Everything else is already implemented with regards to transparency and auditability, but for this specific case if a github alias is not a contact method I would prefer the use of a DL. This is due to the reason that if _something_ is broken for the infrastructure it is better to have wider visibility and as it is meant to be inter-company I cannot guarantee every owner forever will want to post their work e-mail online.

**Comment by BRMcLaren** [comment]
>Having individual owners does seem to make sense based on these arguments. 

To be clear, individual owners is already a hard requirement for automation.

> We could have github aliases on the contacts list(for SIG chairs etc, we only use github ids and not email) and have a DL for an email address.

I am in agreement with this methodology 👍

**Comment by BRMcLaren** [comment]
This has been completed.
